pkg/app/AppCrudOperationService.go

```diff
@@ -284,7 +292,7 @@ func (impl AppCrudOperationServiceImpl) FindAll() ([]*bean.AppLabelDto, error) {
 	return results, nil
 }
 
-func (impl AppCrudOperationServiceImpl) GetAppMetaInfo(appId int) (*bean.AppMetaInfoDto, error) {
+func (impl AppCrudOperationServiceImpl) GetAppMetaInfo(appId int, installedAppId int, installedAppEnvId int) (*bean.AppMetaInfoDto, error) {
```

Comment:
installedAppEnvId is this taking environment id if yes, rename it to environmentId

Reply:
changed with added comment for easier understanding